core/domain/suggestion_registry.py

```diff
@@ -245,7 +262,7 @@ class SuggestionEditStateContent(BaseSuggestion):
     def __init__( # pylint: disable=super-init-not-called
             self, suggestion_id, target_id, target_version_at_submission,
             status, author_id, final_reviewer_id,
-            change, score_category, last_updated):
+            change, score_category, last_updated=None):
```

Comment:
Why the change to use a default arg (and why is it None)?

(Also, these values are supposed to be set at the datastore layer so I'm not sure why they're being passed in here.)

Reply:
I think domain objects can sometimes represent objects that are yet to be put into the datastore, e.g. newly-created objects.

And you're right -- I checked existing domain objects and this does follow the existing convention, so I withdraw my comment. Thanks for explaining!